esx_service/vmdk_ops.py

```diff
@@ -376,8 +376,10 @@ def vol_info(vol_meta, vol_size_info, datastore):
     vinfo[CAPACITY][ALLOCATED] = vol_size_info[ALLOCATED]
     vinfo[LOCATION] = datastore
 
-    if kv.ATTACHED_VM_NAME in vol_meta:
-       vinfo[ATTACHED_TO_VM] = vol_meta[kv.ATTACHED_VM_NAME]
+    if kv.ATTACHED_VM_UUID in vol_meta:
+       vm = findVmByUuid(vol_meta[kv.ATTACHED_VM_UUID])
+       if vm:
+          vinfo[ATTACHED_TO_VM] = vm.config.name
```

Comment:
Yes, it can be removed as the new code will simply ignore it if its there. Updated and pushed the changes.